examples/nidaqmx/analog-input.py

```diff
@@ -79,6 +80,14 @@ def RaiseIfError(response):
         timeout=10.0))
     RaiseIfError(response)
     print(f"Acquired {response.samps_per_chan_read} samples")
+except grpc.RpcError as rpc_error:
+    error_message = rpc_error.details()
+    if rpc_error.code() == grpc.StatusCode.UNAVAILABLE :
```

Comment:
nit: there's an extra space after `UNAVAILABLE` here (and similarly in all the other examples)